### ISSUE TITLE
Not working if script loaded before codemiror

### ISSUE THREAD

**Issue by zortext** [easylogic/codemirror-colorpicker#28]
**Describe the bug**
If the picker script is loaded before codemiror script, color picker does not appear in codemirror.
This is causes some issues if the scripts loaded on-demand for both codemiror and standalone usage.
On some cases standalone picker is loaded before codemiror requested for loading.

**To Reproduce**
Load picker script before codemiror script.
 
**Additional context**
A public  method like "rebuildForCodemirror()"  can solve the issue. Currently following section which is private, initiates picker for code mirror. A public wrapper method for this part can be triggered when needed by using the API

```
 if (CodeMirror) {
        CodeMirror.defineOption("colorpicker", false, function (cm, val, old) {
            if (old && old != CodeMirror.Init) {

                if (cm.state.colorpicker) {
                    cm.state.colorpicker.destroy();
                    cm.state.colorpicker = null;
                }
                // remove event listener
            }
            if (val) {
                cm.state.colorpicker = new ColorView(cm, val);
            }
        });
    }
```

**Comment by easylogic** [comment]
hi @zortext ,

what's colorpicker version ?

**Comment by zortext** [comment]
Latest codemirror-colorpicker/dist/codemirror-colorpicker.min.js

**Comment by easylogic** [comment]
@zortext  thanks. 

i will test it and let you know.

**Comment by easylogic** [comment]
@zortext   

ok, i understood. 😄

**Comment by easylogic** [comment]
@zortext  

i released v1.9.45.

**Comment by easylogic** [comment]
i released v1.9.46 

please see . https://codepen.io/easylogic/pen/yRPZxJ 

you can use `load' function 

`
CodeMirrorColorPicker.load() //  add redefine
CodeMirror.fromTextarea(...);   
`

**Comment by zortext** [comment]
Perfect... Thank you.
